src/main/java/org/spongepowered/api/data/DataManager.java

```diff
@@ -203,30 +161,30 @@
     <T> Optional<DataTranslator<T>> getTranslator(Class<T> objectClass);
```

Comment:
@gabizou @Cybermaxke Is it possible to create a similar method, but with TypeToken? I have problems getting a general translator for a whole class of objects.

```
// public abstract class SomeClass<T extends SomeClass<T>>
// public final class SomeClassTranslator implements DataTranslator<SomeClass<?>>

// Type mismatch: cannot convert from Optional<DataTranslator<SomeClass>> to Optional<DataTranslator<SomeClass<?>>>
Optional<DataTranslator<SomeClass<?>>> translatorBox = Sponge.getDataManager().getTranslator(SomeClass.class);
```